.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: pytorch-widedeep
+
+on:
+  push:
+#    branches: [ $default-branch ]
```

Comment:
@5uperpalo if we do not specify branch, does this imply that all push from all branches will be unit-tested?. 

Just curiosity

Reply:
yes, this implies push event in any branch, I could find only this comment in the documentation: